### PR TITLE
Update link to registration form to 2016

### PR DESCRIPTION
**Pull request by msteinrueck**

### CODE REVIEW

File: _data/wtm_faq.yml [hunk 1]
```diff
@@ -34,7 +34,7 @@
   answer: "Heck yes! This is a great opportunity to network and collaborate with women techies of all levels. Also, this is your chance to bring your cool app idea, website idea, or product idea to life."
 
 - question: "I can’t contain myself. How do I sign up?"
-  answer: "I thought you’d ask! Register here: http://www.meetup.com/Google-Developer-Group-Tucson/events/225462550/"
+  answer: "I thought you’d ask! Register here: <a href='http://www.meetup.com/Google-Developer-Group-Tucson/events/233549567/'>http://www.meetup.com/Google-Developer-Group-Tucson/events/233549567/</a>"
```
Comment (dmsnell) [inline _data/wtm_faq.yml]: How about making the "here" part the link instead of showing the URL?

```html
Register <a href="…">here</a>!
```

**Comment by msteinrueck** [comment]
Makes sense. I realized, if I archive the WTM page, I should archive this as well, since it is included in that page. I don't think I fully understand the directory structure of the CodeForTucson page or why this file is in the "_data" directory. Could you either give me a short introduction or let me know where and under which name I should store the archived version?

**Comment by dmsnell** [comment]
> why [is] this file is in the "_data" directory

good :question: the `data` directory provides a place in **Jekyll** sites where we can store data in a structured way, using [YAML](https://en.wikipedia.org/wiki/YAML), for example. You might notice in [womentechmakers.html](https://github.com/CodeForTucson/codefortucson-site/blob/4764315dcfb3bb71e55a7afea95015ed97f0ba4c/pages/womentechmakers.html#L36) that we load it in as if it were an array. The YAML file provides that array-of-objects with its syntax.

We could have alternatively chosen to include the FAQ list in the front-matter of the women-tech-makers page, but keeping it separate in `data` does a good job separating out the _data_ proper from the view that displays the data and hopefully makes it easier to pop in, like you have done in this PR, and make some edits without touching the site HTML

**Comment by dmsnell** [comment]
> I should archive this as well

this is all subjective. I don't think this one necessitates archiving. FAQs in general tend to be more ephemeral than event pages proper. It looked to me like it was little more than a move from Google to Meetup.com anyway.

File: _data/wtm_faq.yml (hunk 1 above)
Comment (dmsnell) [inline _data/wtm_faq.yml]: @msteinrueck once this is updated I think we'll be ready to merge!

**Comment by dmsnell** [comment]
Closing due to the merge of #83 and #84

**Pull request closed without merging**